src/Akeneo/Connectivity/Connection/back/Infrastructure/Webhook/Client/GuzzleWebhookClient.php

```diff
@@ -88,6 +94,17 @@ public function bulkSend(iterable $webhookRequests): void
                     $webhookRequestLog->setEndTime(microtime(true));
                     $webhookRequestLog->setResponse($response);
 
+                    $pimEvents = array_map(
+                        fn (WebhookEvent $apiEvent) => $apiEvent->getPimEvent(),
+                        $webhookRequestLog->getWebhookRequest()->apiEvents()
+                    );
+                    
+                    $eventsApiRequestSucceeded = new EventsApiRequestSucceeded(
+                        $webhookRequestLog->getWebhookRequest()->webhook()->connectionCode(),
+                        $pimEvents
+                    );
+                    $this->eventDispatcher->dispatch($eventsApiRequestSucceeded);
```

Comment:
```suggestion
                    $this->eventDispatcher->dispatch(new EventsApiRequestSucceeded(
                        $webhookRequestLog->getWebhookRequest()->webhook()->connectionCode(),
                        $pimEvents
                    ));
```
I think this variable was useless :)